src/routes/userRoutes.test.js

```diff
@@ -50,11 +55,7 @@ describe('Testing user routes', () => {
       email: 'hello@world.com'
     }
 
-    db.getModels = jest.fn().mockReturnValue({
-      Accounts: {
-        findOne: jest.fn().mockReturnValue(userAccount)
-      }
-    })
+    mcokFindOne.mockReturnValue(userAccount)
```

Comment:
same as here:
i changed `mcokFindOne.mockReturnValue(userAccount)`
to
`  db.getModels = jest.fn().mockReturnValue({
       Accounts: {
         findOne: jest.fn().mockReturnValue(userAccount)
       }
     })
` let's make an agreement. i think both way worked

Reply:
Both way works. I prefer the way you did. But only in this line, it was more convinient to me to set a container called `mockFindOne` because it is re-mocked later

Reply:
i think use mockFindOne is very precise ; however here there is a typo-- shall be mock not mcok ~~

Reply:
well, since you are being very consistent using mcok, that's good too ;)

Reply:
I fixed